corehq/apps/users/models.py

```diff
@@ -1453,8 +1453,10 @@ def save(self, fire_signals=True, **params):
 
         if fire_signals:
             from .signals import couch_user_post_save
+            from corehq.apps.callcenter.tasks import sync_webuser_usercases_if_applicable
             results = couch_user_post_save.send_robust(sender='couch_user', couch_user=self)
             log_signal_errors(results, "Error occurred while syncing user (%s)", {'username': self.username})
+            sync_webuser_usercases_if_applicable(self._id, spawn_task=True)
```

Comment:
since this is called directly you might as well pass in the user object instead of just the ID. That will save having to fetch it again.